src/hugr/views/sibling.rs

```diff
@@ -41,24 +43,63 @@ pub struct SiblingGraph<'g, Root = Node> {
     _phantom: std::marker::PhantomData<Root>,
 }
 
+/// HugrView trait members common to both [SiblingGraph] and [SiblingMut],
+/// i.e. that rely only on [HugrInternals::base_hugr]
+macro_rules! base_members {
```

Comment:
Uhm, this is iffy but I guess it's ok.
I'd just rename it to `impl_base_members` to convey a bit more info at the callsites.